src/tests/unit/tests/bug-filing/azure-devops/azure-devops-bug-filing-service.test.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+import {
+    AzureDevOpsBugFilingService,
+    AzureDevOpsBugFilingSettings,
+} from '../../../../../bug-filing/azure-devops/azure-devops-bug-filing-service';
+import { SettingsFormProps } from '../../../../../bug-filing/types/settings-form-props';
+import { UserConfigMessageCreator } from '../../../../../common/message-creators/user-config-message-creator';
+import { BugServicePropertiesMap } from '../../../../../common/types/store-data/user-configuration-store';
+
+describe('AzureDevOpsBugFilingServiceTest', () => {
+    let userConfigMessageCreatorMock: IMock<UserConfigMessageCreator>;
+    let props: SettingsFormProps<AzureDevOpsBugFilingSettings>;
+    let projectStub: string;
+    let issueDetailsLocationStub: string;
+
+    const invalidTestSettings = [
```

Comment:
This could use the type... just in case we need to rename any properties on the future.

Reply:
Will add!